language/nodejs/run-tests.md

```diff
@@ -6,19 +6,7 @@ description: How to Build and Run your Tests using Node.js and Mocha frameworks
 
 {% include_relative nav.html selected="4" %}
 
-Testing is an essential part of modern software development. Testing can mean a lot of things to different development teams. There are unit tests, integration tests and end-to-end testing. In this guide we take a look at running your unit tests in Docker.
-
-## Sample application
-
-We’ll use a fairly simple node.js application for this guide. To follow along, clone the following github repository.
-
-```shell
-$ git clone [ADD APPLICATION URL]
-```
-
-Our sample application uses an open source framework called Ronin.js for building simple REST services. It also has a mocks server that will accept JSON and return the same JSON when you send a GET request.
-
-So, for example, if we POST a JSON data structure to the endpoint /services/users and then perform a POST request to the same URI, we will receive the JSON which we first posted.
+Testing is an essential part of modern software development. Testing can mean a lot of things to different development teams. There are unit tests, integration tests and end-to-end testing. In this guide we take a look at running your unit tests in Docker. Let's assume we have defined Mocha tests in a `/test` folder within our application.
```

Comment:
NIT: It should be `test` folder, or `./test` folder to make it clear that it's a relative path to this sub directory.

Reply:
Fixed